hw2/04-background.js

```diff
@@ -1 +1,64 @@
-// Add your code here
+// Stuart Rimel
+const bodyElement = document.getElementsByTagName('body');
+const intervalInputElement = document.getElementById('intervalInput');
+const buttonElement = document.getElementById('input-btn');
+const ALPHA = 0.7;
+let interval = '';
+let intervalID = null;
+
+const getRandomColor = function getRandomColor() {
+  const r = Math.floor(Math.random() * 255);
+  const g = Math.floor(Math.random() * 255);
+  const b = Math.floor(Math.random() * 255);
+  return `rgba(${r}, ${g}, ${b}, ${ALPHA})`;
+};
+
+const setRandomBackgroundColor = function setRandomBackgroundColor() {
+  bodyElement[0].style.backgroundColor = getRandomColor();
+};
+
+const handlePageLoad = function handlePageLoad(event) {
```

Comment:
_Style_ - `event` here is never used ([13.8](https://github.com/airbnb/javascript#variables--no-unused-vars)).

Reply:
fixed in updated PR